### DISCUSSION TITLE
Flashing says it's working but I can't connect

### DISCUSSION THREAD

**Discussion by Vulpus-Technica** [darthcloud/BlueRetro#506] [Q&A]
It says I flashed the program successfully, but I can't get a wireless signal.  I'm using a bare esp32-WROOM-32, and a PS4 controller.  It worked when I did this about a month ago, but now I have this issue.  I can't get any wireless connection.

![Screenshot 2022-12-18 232206](https://user-images.githubusercontent.com/17378287/208347171-8fa72ed8-d581-4efa-a4c2-ed4b01cccbad.png)

**Comment by darthcloud** [comment]
You need to check the checkbox on the left of the files name, it wont do anything otherwise.

https://github.com/darthcloud/BlueRetro/wiki/Flashing-firmware-Windows-10
